Parsers/RemoteDesktopServices-RdpCoreTS-parser.txt

```diff
@@ -0,0 +1,116 @@
+// KQL RemoteDesktopServices-RdpCoreTS Event Parser
+// Last Updated Date: May 23, 2020
+//
+// Purpose : 
+// This parser aims to help collecting data from Windows VM through the Microsoft-Windows-RemoteDesktopServices-RdpCoreTS/Operational events channel.
+// This might be usefull to detect success bruteforce attacks on RDP.
+// Note that this parser was initially created to monitor EventID 140 (RDP failed auth.). Thus, at the moment, not all EventID are supported.
+// Don't hesitate to update it or to ask for an update.
+//
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias (e.g. RDPCoreTS).
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. RDPCoreTS | where EventID == 140).
+//
+// References : 
+// Using functions in Azure monitor log queries : https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/functions
```

Comment:
We recently updated all our doc links to remove locale en-us. 
Can you just replace this url without locale :  https://docs.microsoft.com/azure/azure-monitor/log-query/functions